conf/messages.en

```diff
@@ -47,19 +47,27 @@ profileSetup.ukims.h2 = UK Internal Market Scheme (UKIMS)
 profileSetup.p1 = The trader who is registering for a Trader Goods Profile must be UKIMS registered. You’ll need your UKIMS numbers to set up your profile.
 profileSetup.p2 = If you’ve applied to UKIMS, HMRC will have sent you a letter with your UKIMS authorisation number.
 profileSetup.p3 = If you are not UKIMS registered, you can check here to find out {0}.
-profileSetup.p3.linkText = who can apply to UKIMS
+profileSetup.p3.linkText = <a target="_blank" href="https://www.gov.uk/guidance/apply-for-authorisation-for-the-uk-internal-market-scheme-if-you-bring-goods-into-northern-ireland" class="govuk-link">who can apply to UKIMS</a>
```

Comment:
This is probably a result of when you split off the branch, but ProfileSetup is using the link component now (and next two)